examples/custom_file_format/main.rs

```diff
@@ -0,0 +1,69 @@
+use config::{Config, Format, Map, Value, ValueKind, FileStoredFormat};
```

Comment:
To avoid the rustfmt new line requirement, `config::File` change to `File`, import that too:

```suggestion
use config::{Config, File, FileStoredFormat, Format, Map, Value, ValueKind};
```